WordPress/src/main/res/values-v27/styles.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="WordPress" parent="Base.Wordpress">
+        <item name="android:navigationBarColor">@color/white</item>
```

Comment:
```suggestion
        <item name="android:navigationBarColor">@android:color/white</item>
```

Can we change this to `@android:color/white`? There is no resource white in `colors.xml` so using `@color/white` would be pulling from another source or library.

Reply:
👍 Fixed in d11e529a33c8401d4a1f955723013d2a460e3e38